it.json

```diff
@@ -83,6 +83,11 @@
                 "trigger" : "language",
                 "title" : "Scegli lingua",
                 "currentLanguage" : "Stai attualmente usando [language]."
+                "error" : "Errore durante il download del file delle traduzioni",
+                "authors" : "Tradotto da:",
+                "cta" : "Clicca per cambiare a [language]",
+                "switching" : "Scaricando [language]...",
+	            "finished" : "La tua lingua e' stata cambiata!"
```

Comment:
Wrong indentation. Please remove 4 spaces so that this translation is on the same level as the other translations above it.